modules/core/24-host/client_keys.go

```diff
@@ -40,3 +40,8 @@ func FullConsensusStateKey(clientID string, height exported.Height) []byte {
 func ConsensusStateKey(height exported.Height) []byte {
 	return []byte(ConsensusStatePath(height))
 }
+
+// CounterpartyKey returns the store key for the counterparty information.
+func CounterpartyKey() []byte {
```

Comment:
given that this is in the private store, I think maybe we should be putting private keys into the 02-client/types?